### ISSUE TITLE
Is there a page for me to see all "Commentary & Analysis" articles?

### ISSUE THREAD

**Issue by danblah** [ResetNetwork/recoding-tech#85]
If so, where is it and if not, should there be one?

**Comment by harumhelmy** [comment]
There is not a page where you can do this! I think it might make sense to have, looking forward to hearing what others think.

**Comment by rudietuesdays** [comment]
I also think it makes sense, but we'd need a design for where to link to it and the layout for the page before we can code it out.

**Comment by gt150** [comment]
This is fairly low priority for me. First, we don't have a ton of articles yet. That might change deepening upon greater participation from Reset country teams, but right now there'd be very little there. Second, my design intention for the site was to avoid the typical news site of building everything around articles and structure things around the topics (issues, policy, government, and companies) along with the law and regulation tracker items and then build the information around those items (whether they come from recoding or other sources). At the end of the day, my goal was to direct users to the best information to understand a given topic, whether that's a recoding article or someone else article, report, policy paper, etc. I do think we need to make some adjustments to the homepage to be able to list more content (like articles) and other updates so that it looks more filled-in and gives us greater space to feature new/important content from the site.
